tests/unit/amo/components/TestAddonDetail.js

```diff
@@ -313,6 +322,30 @@ describe('AddonDetail', () => {
     expect(rootNode.querySelector('.InstallButton-switch input').disabled).toBe(true);
   });
 
+  it('enables a theme preview for non-enabled add-ons', () => {
+    const rootNode = renderAsDOMNode({
+      addon: {
+        ...fakeAddon,
+        type: ADDON_TYPE_THEME,
+      },
+      status: UNKNOWN,
+    });
+    const button = rootNode.querySelector('.AddonDetail-theme-header-label');
+    expect(button).toBeTruthy();
```

Comment:
This is a bit nit-picky but I've been trying to always at least separate tests into two sections with a newline: one is actions to set up the tests, the other is a list of expectations we're asserting. So this would simply become:

```js
it('enables a theme preview for non-enabled add-ons', () => {
  const rootNode = renderAsDOMNode({
    addon: {
      ...fakeAddon,
      type: ADDON_TYPE_THEME,
    },
    status: UNKNOWN,
  });
  const button = rootNode.querySelector('.AddonDetail-theme-header-label');

  expect(button).toBeTruthy();
});
```

I think that really helps highlight what we're testing and makes the tests less likely to be chunks of code a future developer glazes over. I find that whitespace really helps illustrate intent and importance.